CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Please check our [developers guide](https://gitlab.com/tokend/developers-guide)
 for further information about branching and tagging conventions.
 
 ## [Unreleased]
+### Fixed
```

Comment:
<img width="394" alt="image" src="https://user-images.githubusercontent.com/49021956/205959335-73b9c298-9896-4f79-a074-df5cb91d0feb.png">
lets update the type sequence and change it from 'fixed' to 'changed'